tasks/dataset-split/task.yaml

```diff
@@ -0,0 +1,16 @@
+param_groups:
```

Comment:
Templates should also have a few more fields before param_goups:
id:
name:
is_active:
project_type: